plugins/memcached.yaml

```diff
@@ -18,17 +17,32 @@ parameters:
     default: end
 
 # Set Defaults
-# {{$file_path := default "/var/log/memcached.log*" .file_path}}
 # {{$start_at := default "end" .start_at}}
 
 # Pipeline Template
 pipeline:
-  - id: memcached_input
-    type: file_input
+  - id: memcached_journald_input
+    type: journald_input
     include:
-      - {{ $file_path }}
+      - {{ .memcached_journald_log_path }}
     start_at: {{ $start_at }}
     labels:
       log_type: memcached
       plugin_id: {{ .id }}
-    output: {{.output}}
+
+  - id: memcached_filter
```

Comment:
If we add an option to journald to allow filtering log messages, we can do this much more efficiently. No change here for now, but I'll make a stanza ticket